bin/latex.py

```diff
@@ -90,7 +90,7 @@ def get_tl(problem):
 
     if 'print_timelimit' in contest_yaml():
         print_tl = contest_yaml()['print_timelimit']
-    elif hasattr(config.args, 'no_timelimit'):
+    elif config.args.no_timelimit is not None:
```

Comment:
`is not None` isn't really needed right?  `false` and `None` should be equivalent (if it was possible to set the flag to False at all at some point in the future.)

Reply:
You're right, `None` defaults to `False` anyway, so I inlined it :slightly_smiling_face: (57be5a8)